admin/migrate-to-deb.sh

```diff
@@ -0,0 +1,124 @@
+if [[ ! -f /etc/os-release ]] || ! grep -q 'NAME="Tails"' /etc/os-release; then
+    error_exit "This script must be run on Tails.\n\nCurrent system is not Tails."
+fi
+echo "- Running on Tails"
+
+# Check Tails version >= 7
+tails_version=$(grep '^VERSION=' /etc/os-release | cut -d= -f2 | tr -d '"')
+tails_major_version=$(echo "$tails_version" | cut -d. -f1)
+if (( tails_major_version < 7 )); then
+    error_exit "This migration requires Tails 7 or later.\n\nCurrent version: $tails_version\n\nPlease upgrade Tails before migrating."
+fi
+echo "- Tails version: $tails_version"
+
+# Check if old config directory exists
+OLD_CONFIG_DIR="$HOME/Persistent/securedrop/install_files/ansible-base"
+if [[ ! -d "$OLD_CONFIG_DIR" ]]; then
+    error_exit "Old configuration directory not found.\n\nExpected: $OLD_CONFIG_DIR\n\nThis script is for migrating existing installations only."
+fi
+echo "- Old config directory found: $OLD_CONFIG_DIR"
+
+# Verify root script exist
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_SCRIPT="$SCRIPT_DIR/configure-tails-persistence.sh"
+if [[ ! -f "$ROOT_SCRIPT" ]]; then
+    error_exit "Helper script not found.\n\nExpected: $ROOT_SCRIPT"
+fi
+
+# Run the root script
+echo "Configuring Tails persistence (requires password)..."
+if ! pkexec bash "$ROOT_SCRIPT"; then
+    error_exit "Failed to configure Tails persistence."
+fi
+
+# Verify installation
+if ! command -v /usr/bin/securedrop-admin >/dev/null 2>&1; then
+    error_exit "Package installed but securedrop-admin command not found.\n\nInstallation may have failed."
+fi
+echo "- securedrop-admin command is available"
+
+NEW_CONFIG_DIR="$HOME/.config/securedrop-admin"
+
+# Copy site-specific config
+SITE_SPECIFIC_FILE="$OLD_CONFIG_DIR/group_vars/all/site-specific"
+if [[ -f "$SITE_SPECIFIC_FILE" ]]; then
+    cp "$SITE_SPECIFIC_FILE" "$NEW_CONFIG_DIR/"
+    echo "- Migrated: site-specific"
+
+    # Parse site-specific for GPG public key filenames and copy them
+    ossec_key=$(grep '^ossec_alert_gpg_public_key:' "$SITE_SPECIFIC_FILE" | awk '{print $2}' | tr -d "'\"")
+    if [[ -n "$ossec_key" && "$ossec_key" != "''" && -f "$OLD_CONFIG_DIR/$ossec_key" ]]; then
+        cp "$OLD_CONFIG_DIR/$ossec_key" "$NEW_CONFIG_DIR/"
+        echo "- Migrated: $ossec_key (OSSEC GPG public key)"
+    elif [[ -n "$ossec_key" && "$ossec_key" != "''" ]]; then
+        echo "! Not found (skipping): $ossec_key (OSSEC GPG public key)"
+    fi
+
+    securedrop_key=$(grep '^securedrop_app_gpg_public_key:' "$SITE_SPECIFIC_FILE" | awk '{print $2}' | tr -d "'\"")
+    if [[ -n "$securedrop_key" && "$securedrop_key" != "''" && -f "$OLD_CONFIG_DIR/$securedrop_key" ]]; then
+        cp "$OLD_CONFIG_DIR/$securedrop_key" "$NEW_CONFIG_DIR/"
+        echo "- Migrated: $securedrop_key (SecureDrop GPG public key)"
+    elif [[ -n "$securedrop_key" && "$securedrop_key" != "''" ]]; then
+        echo "! Not found (skipping): $securedrop_key (SecureDrop GPG public key)"
+    fi
```

Comment:
There is a 3rd optional GPG pubkey used for configuring journalist alerts (this is distinct from the ossec key) - if it's configured and not present it's also a fail state. If it's not present in `site-specific` tho, that's fine.